inference-engine/ie_bridges/java/cpp/tensor_desc.cpp

```diff
@@ -0,0 +1,223 @@
+#include <inference_engine.hpp>
+
+#include "openvino_java.hpp"
+#include "enum_mapping.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_TensorDesc_GetTensorDesc(JNIEnv *env, jobject obj, jint precision, jintArray dims, jint layout)
+{
+    static const char method_name[] = "GetTensorDesc";
+    try
+    {
+        auto n_precision = precision_map.find(precision);
+
+        if (n_precision == precision_map.end())
+            throw std::runtime_error("No such precision value!");
```

Comment:
Can you please experiment if 
```cpp
auto n_precision = precision_map.at(precision);
```
with invalid `precision` will throws reasonable exception message? In this way it will simplify our code a lot:

```
auto n_precision = precision_map.at(precision);
auto n_layout = layout_map.at(layout);
```

Reply:
There is no exception about wrong precision from this method.
![image](https://user-images.githubusercontent.com/32199725/86015326-1a3fc100-ba2a-11ea-8eda-49088aa3ed72.png)

Reply:
Example for setPrecision metod
![image](https://user-images.githubusercontent.com/32199725/86015627-7a366780-ba2a-11ea-8499-eb31fff12e8b.png)